docker/indexer/config/config.go

```diff
@@ -0,0 +1,62 @@
+// Package config provides functionality to load configurations
+package config
+
+import (
+	"context"
+	"io"
+	"strings"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+
+	"github.com/golang/protobuf/proto"
+
+	log "github.com/golang/glog"
+	pb "github.com/google/osv.dev/docker/indexer/proto"
+)
+
+// Load loads the repository configurations from the provided bucket.
+func Load(ctx context.Context, cfgBucket *storage.BucketHandle) ([]*pb.Repository, error) {
+	var repos []*pb.Repository
+	nameTracker := make(map[string]bool)
+	iter := cfgBucket.Objects(ctx, nil)
+	for {
+		attrs, err := iter.Next()
+		if err == iterator.Done {
+			break
+		}
+		if err != nil {
+			return nil, err
+		}
+
+		if strings.HasSuffix(attrs.Name, "textproto") {
```

Comment:
nit: invert condition and continue instead to avoid unnecessary indentation here for the rest of the code here.